src/firefly/js/visualize/saga/ImageMetaDataWatcher.js

```diff
@@ -37,7 +37,7 @@ export function* watchImageMetaData({viewerId}) {
     var tbl_id;
     var paused= true;
     while (true) {
-        const action= yield take([TABLE_NEW,TABLE_SELECT,TABLE_HIGHLIGHT, TABLE_UPDATE, TABLE_REMOVE,
+        const action= yield take([TABLE_SELECT,TABLE_HIGHLIGHT, TABLE_UPDATE, TABLE_REMOVE,
```

Comment:
if you are watching TABLE_UPDATE, then TABLE_LOADED is not needed, since UPDATE is also fired on first load as well.